src/ui/tabview/TabPage.qml

```diff
@@ -27,11 +27,16 @@ import ".."
 Item {
     id: tabPage
 
+    // binding. should be set from outside
```

Comment:
Is this comment still relevant? I don't understand what it means.

Reply:
It's relevant. It means that activeTab property itself doesn't do anything. It should be a functions assigned by another component. This function should return an activeTab. I did this to avoid dependency on TabController

Reply:
Ah, all right.